tutorials/text/mathtext.py

```diff
@@ -66,6 +66,15 @@
 
     \alpha_i > \beta_i
 
+Note that to make multi-letter subscripts or superscripts displayed correctly,
```

Comment:
This is not really a note, per se, so I would do away with that phrasing, and just say "To display multi-letter ..."

Reply:
Fair enough. I committed this change. Do I need to do any other thing for this PR to proceed? Because it merged before this change.